examples/plot_estimate_global_reject.py

```diff
@@ -39,6 +39,9 @@
 # Now we get the rejection dictionary
 
 from autoreject import get_rejection_threshold  # noqa
+
+# We could use the `decim` parameter to only take every nth time slice
+# this would speed up the computation time at the expense of accuracy
```

Comment:
not necessarily unless you have very peaky artifacts ...

Reply:
agreed, I updated my comment to reflect this.